platforms/hyperledger-fabric/charts/create_channel/templates/create_channel.yaml

```diff
@@ -146,11 +146,22 @@ spec:
             echo "Channel ${CHANNEL_NAME} is already created."
           else
             echo "Creating Channel ${CHANNEL_NAME}"
+            found=`echo $NETWORK_VERSION | grep -c 1.4`
```

Comment:
Instead check Lakshya's code that is already merged to develop. The if condition is in the chart/template itself.